connectors/src/lib/models/slack.ts

```diff
@@ -311,6 +311,7 @@ export class SlackBotWhitelistModel extends Model<
   declare updatedAt: CreationOptional<Date>;
   declare botName: string;
   declare groupIds: string[];
+  declare whitelistType: "summon_agent" | "index_messages";
```

Comment:
NIT: maybe we can use `SlackbotWhitelistType`?

Reply:
We absolutely should. Thanks for spotting !